conmon/conmon.c

```diff
@@ -126,6 +127,7 @@ static GOptionEntry opt_entries[] =
   { "cid", 'c', 0, G_OPTION_ARG_STRING, &opt_cid, "Container ID", NULL },
   { "cuuid", 'u', 0, G_OPTION_ARG_STRING, &opt_cuuid, "Container UUID", NULL },
   { "runtime", 'r', 0, G_OPTION_ARG_STRING, &opt_runtime_path, "Runtime path", NULL },
+  { "restore", 0, 0, G_OPTION_ARG_NONE, &opt_restore, "Restore a container from a checkpoint", NULL },
```

Comment:
Do you need to have an argument for the checkpoint to restore from?

Reply:
The checkpoint to restore from is defined by the container ID and the bundle.